src/schemas/21P-530/schema.js

```diff
@@ -34,6 +34,42 @@ let schema = {
   type: 'object',
   additionalProperties: false,
   definitions: {
+    address: {
+      type: 'object',
+      properties: {
+        street: {
+          type: 'string'
+        },
+        aptNum: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 5
```

Comment:
A lot of the validation in here is really restrictive, and I realize that's because of the form boxes, but when a user is normally filling out the form they can write outside the boxes or put their address on another page. They don't have that option in our form with these kinds of limits.